docker/docker-settings.toml

```diff
@@ -0,0 +1,8 @@
+[lora]
```

Comment:
Everything here should be in `dev-environment/dev-settings.toml`. From a quick glance at `backend/mora/default-settings.toml`, I dont think anything needs to be set in this file for now.